identity-apps-core/apps/recovery-portal/src/main/webapp/password-recovery-otp.jsp

```diff
@@ -0,0 +1,315 @@
+<%--
+  ~ Copyright (c) 2016-2024, WSO2 LLC. (https://www.wso2.com).
```

Comment:
Since this file is new to the identity-apps repo, let's only keep 2024 here.

```suggestion
  ~ Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
```

Reply:
fixed with https://github.com/wso2/identity-apps/pull/5817/commits/c4711e788c5f3c08e1e2906f20e2217b381f34e9